petisco/event/consumer/infrastructure/rabbitmq_event_consumer.py

```diff
@@ -142,7 +151,15 @@ def rabbitmq_consumer(
                 )
                 result = Failure(EventChaosError())
             else:
-                result = handler(event)
+                params = inspect.getfullargspec(handler).args
```

Comment:
Didn't know this. Cool!

Reply:
We're already using it, here https://github.com/alice-biometrics/petisco/blob/master/petisco/controller/controller_handler.py#L136